ardere/step_functions.py

```diff
@@ -127,6 +135,58 @@ def _validate_plan(self):
         # Replace our event with the validated
         self.event = data
 
+    def _create_dashboard(self, grafana_url, dashboard_options):
+        # type: (str, Dict[str, any]) -> str
+        """Create the dashboard in grafana"""
+        s3_filename = dashboard_options["filename"]
+        s3 = self.boto.resource('s3')
+        dash_file = s3.Object(
+            os.environ["metrics_bucket"],
+            s3_filename
+        )
+        file_contents = dash_file.get()['Body'].read().decode('utf-8')
+        dash_contents = json.loads(file_contents)
+        dash_contents["title"] = dashboard_options["name"]
+        dash_contents["id"] = None
+
+        response = requests.post(grafana_url+"/api/dashboards/db",
+                                 auth=('admin', 'admin'),
```

Comment:
Nope, this goes into a config or env or anything else. 

Also, probably want to change this password.